dashboard/app/views/pd/session_attendance/upgrade_account.html.haml

```diff
@@ -0,0 +1,23 @@
+%h1
+  Upgrade to a teacher account
+
+%p
+  Your account is a student account. We need to upgrade your account to a teacher account to proceed.
+
+- if @email_mismatch
+  #error_explanation
+    %h2 Error
+    %p
+      - if @email.blank?
+        Email is required
+      - else
+        Email must match your login. If you want to use this email instead, first update it in
+        = link_to 'account settings', "/users/edit?user_return_to: #{request.url}"
```

Comment:
as we discussed offline, I think this return param won't actually work just yet; have you made a note somewhere for either you or I to add that functionality?